metrics/report/README.md

```diff
@@ -0,0 +1,73 @@
+* [cloud-native-setup metrics report generator](#cloud-native-setup-metrics-report-generator)
+   * [Data gathering](#data-gathering)
+   * [Report generation](#report-generation)
+   * [Debugging and development](#debugging-and-development)
+
+# cloud-native-setup metrics report generator
+
+The files within this directory can be used to generate a 'metrics report'
+for Kubernetes.
+
+The primary workflow consists of two stages:
+
+1) Run the provided report metrics data gathering scripts on the system(s) you wish
+to analyze.
+2) Run the provided report generation script to analyze the data and generate a
+report file.
+
+## Data gathering
+
+Data gathering is provided by the `grabdata.sh` script. When run, this script
+executes a set of tests from the `cloud-native-setup/metrics` directory. The JSON results files
+will be placed into the `cloud-native-setup/metrics/results` directory.
+
+Once the results are generated, create a suitably named subdirectory of
+`tests/metrics/results`, and move the JSON files into it.
+
+Repeat this process if you want to compare multiple sets of results. Note, the
+report generation scripts process all subfolders of `tests/metrics/results` when
+generating the report.
+
+You can restrict the subset of tests run by `grabdata.sh` via its commandline parameters:
+
+| Option | Description |
+| ------ | ----------- |
+| -a | Run all tests (default) |
+| -d | Run the density tests |
```

Comment:
here the option is -d for density, below it's -s for scaling. This one should be updated to -s I believe.

Reply:
Ah, thanks. Yep, I did a s/density/scaling/ but looks like I missed a few...